lib/active_merchant/billing/gateways/commerce_hub.rb

```diff
@@ -0,0 +1,309 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class CommerceHubGateway < Gateway
+      self.test_url = 'https://cert.api.fiservapps.com/ch'
+      self.live_url = 'https://prod.api.fiservapps.com/ch'
+
+      self.supported_countries = ['US']
+      self.default_currency = 'USD'
+      self.supported_cardtypes = %i[visa master american_express discover]
+
+      self.homepage_url = 'https://developer.fiserv.com/product/CommerceHub'
+      self.display_name = 'CommerceHub'
+
+      STANDARD_ERROR_CODE_MAPPING = {}
+
+      SCHEDULED_REASON_TYPES = %w(recurring installment)
+      ENDPOINTS = {
+        'sale' => '/payments/v1/charges',
+        'authonly' => '/payments/v1/charges',
+        'capture' => '/payments/v1/charges',
```

Comment:
Since all three of these are the same endpoint, should they be consolidated into one?

Reply:
Good call out!